src/main/java/edu/harvard/iq/dataverse/search/SearchIncludeFragment.java

```diff
@@ -291,6 +294,7 @@ public void search(boolean onlyDataRelatedToMe) {
                 /**
                  * @todo centralize this into SearchServiceBean
                  */
+                if (!isfilterQueryAlreadyInMap(filterDownToSubtree))
                 filterQueriesFinal.add(filterDownToSubtree);
```

Comment:
The lack of curly braces (preferred) or indentation of the code in the "if" is confusing.

Reply:
re-did curly braces here.

Reply:
I no longer see the confusing "if". Thanks.